### DISCUSSION TITLE
Make an array of non-integer elements in Python pandas

### DISCUSSION THREAD

**Discussion by Adrij1993** [ggml-org/llama.cpp#157] [Q&A] [answered]
Hello all,
I am Adrij from India. I am working on a data analysis work. I will try to explain the problem that I a facing in a few words. 
Say I have a column of 500 rows in which there are some string values. I want to replace all these string values with 0. If there is only type of string value, I can do the manual check & set a code for converting the string to 0. But if there are more than 1 string type, it is very difficult to check individual string element. How can I make an array which will select all those string elements? From the array, later the code can check for each elements in the column row by row, wherever it finds one the string will be zero.

Please suggest your opinion.

**Comment by v3ss0n** [comment, marked as answer]
What this has to do with this project ?

**Reply by Adrij1993** [reply]
Actually there are weather parameter calculation from the raw data. Now the data has two columns filled with string "E4R" in some rows. I solved the issue by code sating that whenever any row element is "E4R" instead of a numerical value, the row element is set to 0. Now suppose if I had any other string instead of "E4R", say "AB" or "HO" or any other string element which I might miss in a 3000 by 25 (row by column) data. How to automatically let the code determine whether a row element is having any string element & convert to 0.

**Reply by v3ss0n** [reply]
Are you a bot? what you are saying dosen't make any sense.

**Comment by Adrij1993** [comment]
How? Please suggest.
